src/MicrosoftTeams.ts

```diff
@@ -2119,4 +2119,43 @@ namespace microsoftTeams {
       ]);
     }
   }
+
+  /**
+   * @private
+   * Hide from docs
+   * --------
+   * Information about all members in a chat
+   */
+  export interface ChatMembersInformation {
+    members: ChatMember[];
+  }
+
+  /**
+   * @private
+   * Hide from docs
+   * --------
+   * Information about a chat member
+   */
+  export interface ChatMember {
```

Comment:
As mentioned in the WebClient PR, I think it might be good to name this something more generic (e.g. ThreadMember or simply User).

Reply:
Changed to ThreadMember